docs/content/release_notes/whats_new.md

```diff
@@ -1,57 +1,13 @@
-# What's new in 1.11.0
+# What's new in 1.11.1
 
-IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detailed in the following section.
+IBM® block storage CSI driver 1.11.1 introduces the enhancements that are detailed in the following section.
 
-**General availability date:** 23 January 2023
+**General availability date:** 8 May 2023
 
-## New dynamic volume group support
+## Supported orchestration platforms for deployment
 
-The new volume group support allows dynamic management of the content of the groups. This feature is used by policy-based replication, introduced IBM Spectrum Virtualize 8.5.2 release.
```

Comment:
this deleted section includes some very important info for the users, I don't  think we should delete it entirely.
we should move somewhere else

Reply:
ok cool